amplitude_python_sdk/v2/ChartAnnotationsApiClient.py

```diff
@@ -0,0 +1,79 @@
+"""Client implementation for the Amplitude V2 Annotations API."""
+
+from typing import Optional, Dict, Any
+
+import requests
+
+from .models.charts import ChartAnnotationsV2
+from ..common.exceptions import AmplitudeAPIException
+from ..common.utils import return_or_raise
+
+
+class ChartAnnotationsApiClient:  # pylint: disable=too-few-public-methods
+    """
+    See <https://developers.amplitude.com/docs/chart-annotations-api> for documentation.
+    """
+
+    def __init__(
+        self, api_key: str,
+        api_endpoint: str = "https://api2.amplitude.com/2/annotations",
+        annotation_id: Optional[str] = None
```

Comment:
Not sure if `annotation_id` can be inserted after instantiation

Reply:
I think the annotation ID should be a method parameter rather than a class parameter (see above). We can add a method like `add_chart_annotations` on `AmplitudeAPIV2Client` instead to facilitate it.

Reply:
done